TickSpec.Tests/FeatureParserTest.fs

```diff
@@ -16,20 +16,30 @@ let private verifyParsing (lines: string[]) (expected: FeatureSource) =
 let private verifyLineParsing (lines: string[]) (expected: LineType list) =
     let lineParsed =
         lines
-        |> Seq.map (fun line ->
+        |> Seq.mapi (fun lineNumber line -> (lineNumber + 1, line))
+        |> Seq.map (fun (lineNumber, line) ->
             let i = line.IndexOf("#")
-            if i = -1 then line
-            else line.Substring(0, i)
+            if i = -1 then lineNumber, line
+            else lineNumber, line.Substring(0, i)
         )
-        |> Seq.filter (fun line -> line.Trim().Length > 0)
-        |> Seq.scan (fun (_, lastLine) line ->
-            let parsed = parseLine (lastLine, line)
+        |> Seq.scan (fun prevLine (lineNumber, lineContent) ->
+            let lastParsedLine =
+                match prevLine with
+                | Line (_, _, prevLineType) -> prevLineType
```

Comment:
```suggestion
                | Line (_, _, prevLineType)
```

Reply:
Good suggestion, did it also in the BlockParser.